platforms/android/library/src/main/java/io/element/android/wysiwyg/inputhandlers/InterceptInputConnection.kt

```diff
@@ -322,7 +323,12 @@ internal class InterceptInputConnection(
         beginBatchEdit()
         editable.removeFormattingSpans()
         editable.replace(0, editable.length, charSequence)
-        setComposingRegion(compositionStart, compositionEnd)
+        val start = compositionStart.coerceIn(0, editable.length)
+        val end = compositionEnd.coerceIn(0, editable.length)
+        val newComposition = editable.substring(start, end)
+        if (newComposition.isEmpty() || !newComposition.isDigitsOnly()) {
```

Comment:
The case where `isEmpty() == true` is already covered by `!isDigitsOnly()` so it could be simplified. Did you mean `isBlank() || !isDigitsOnly()`?

Reply:
Actually, it's not. The tests were failing before I added this condition. `isDigitsOnly` will go through the different codepoints in the string and when one is not a digit return false, otherwise return true after checking all. Since there are no codepoints, it will return that all characters (0 in this case) are digits.

Reply:
Ah OK, my mistake!

Reply:
No worries, I thought the same at first and it took me a while to figure out what was wrong 😅 . It's super counter-intuitive.